go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/posthog/posthog-go v1.2.19
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.9.0
```

Comment:
Can you run `go mod tidy` to update the `go.sum` file along side this new dependency?

Reply:
I ran `go mod tidy` but there were no changes. It looks like you added testify yesterday.

https://github.com/open-sauced/pizza-cli/blob/eb6946f7a4f61a97b5d953f13d83175d0bb0d9a9/go.sum#L123-L126